docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Controller Deployment
+sidebar_label: Controller
+id: deploy
+---
+
+import LinuxPackageRepo from '/docs/_linux-package-repo.mdx';
+import CliLogFormats from '/docs/_cli-log-levels-and-formats.md';
+
+This article is about deploying a Ziti Controller as a Linux system service. [The controller introduction](/learn/introduction/03-components.md#openziti-controller) introduces the architectural concepts used in this guide.
+
+## Installation
+
+The controller package `openziti-controller` provides a systemd service unit and optional bootstrapping. The package depends on the `openziti` package which provides the `ziti` CLI. The easiest way to install both is the cross-platform install script for RPM and Debian distributions.
+
+```text
+curl -sS https://get.openziti.io/install.bash \
+| sudo bash -s openziti-controller
+```
+
+<LinuxPackageRepo />
+
+## Bootstrapping
+
+You can opt-in to generating a PKI, config, and database by installing interactively with prompts or by setting these values.
+
+1. In `/opt/openziti/etc/controller/service.env` (the answer file for the service)
+    1. Set `ZITI_BOOTSTRAP=true`
+1. In `/opt/openziti/etc/controller/bootstrap.env` (the answer file for bootstrapping the controller)
+    1. Set `ZITI_CTRL_ADVERTISED_ADDRESS` to the FQDN of the controller
+    1. Set `ZITI_PWD` to the desired management password for user `admin`. You may delete this after bootstrapping.
+
+### Generating a Public Key Infrastructure

```

Comment:
This would already be accomplished based on the prior comment. This is important, and they need to understand that this all exists, but it's in the scope of backing up this PKI by offlining the root key(s) and when and how to rotate important certs. Any optional type stuff should be reseved for a reference type document. Let's tell them the 95% install case here. The `ZITI_BOOTSTRAP_PKI` stuff should be removed. Instead, we should just tell them here's what we're doing and why.

Rotating the controller's server cert on the user's behalf should be just a presumption that happens on some interval and this should just tell them that it's going to happen and on what interval to expect it to happen.

Using `ALT SERVER CERTS` should be mentioned in here, particularly in relation to zrok/browzer.

Does the signing intermediate need to get rotated? How long will it be valid for? That's should all be captured here for people to be aware of.

Reply:
issue for changing controller leaf cert renewal from startup to interval: https://github.com/openziti/ziti/issues/1994

Reply:
issue for supporting alt server certs: https://github.com/openziti/ziti/issues/2001 (is linked to documentation counterpart issue)